OrleansDashboard.Core/Metrics/GrainProfilerOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace OrleansDashboard.Metrics
+{
+    public sealed class GrainProfilerOptions
+    {
+        public bool TraceAlways { get; set; }
```

Comment:
I wonder if the default behaviour should be set to `true`, so this is a non-breaking change. I'm not 100% sure which should be the preferred default, so curious to hear other people's thoughts.

Reply:
If we add it to 4.0 only I do not see a problem, because it is not backwards compatible anyway .

Reply:
ok, let's do that.